textbasedgame/Program.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+
+namespace textbasedgame
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // assign variables
+            string ch1;
```

Comment:
In general, I would say it is best to declare and use variables closer to where they are relevant. We are not doing JavaScript in this class, so no need to do this.